osquery/dispatcher/scheduler.cpp

```diff
@@ -239,6 +245,13 @@ void SchedulerRunner::start() {
       break;
     }
   }
+
+  // Scheduler ended.
+  if (!interrupted()) {
+    LOG(INFO) << "The scheduler ended after " << FLAGS_schedule_timeout
```

Comment:
Why use `FLAGS_schedule_timeout` here? Wouldn't it be better to write out `_timeout` so we know how long it actually ran for, as opposed to the desired timeout?